packages/liferay-js-insights/README.md

```diff
@@ -0,0 +1,69 @@
+# liferay-js-insights
+
+> A simple dependency analysis reporter. It scans ES6 modules for imports and generate a report on used external packages.
+
+## Installation
+
+Either:
+
+-   Install globally: `yarn global add liferay-js-insights`; or:
+-   Use the latest without installing: `npx liferay-js-insights` (see below for usage).
+
+## Usage
+
+```
+liferay-js-insights [option...]
+
+Options:
+  --airtable    [optional: dumps the analysis to a remote airtable installation]
+  --apiKey      [optional: airtable API KEY]
+  --baseKey     [optional: airtable base]
```

Comment:
I suspect all airtable-related options should be namespaced (eg. `--airtableApiKey`, `--airtableBaseKey`). You could also potentially drop the `--airtable` option then because the presence of the others could imply it?

Reply:
👍